source/api/auth/github.ts

```diff
@@ -69,9 +69,7 @@ export const generateAuthToken = async (req: Request, res: Response, ___: NextFu
 
   const authToken = createPerilUserJWT({ name: user.name, avatar_url: user.avatar_url }, installations)
   // TODO: add max age
-  // TODO: This needs to change when we have production vs staging
   res.cookie("jwt", authToken, { domain: ".peril.systems" })
```

Comment:
you have access to the same installations, because you're not using staging GitHub